src/Microsoft.AspNetCore.Razor.TagHelperTool/RunCommand.cs

```diff
@@ -32,12 +32,15 @@ private int Execute(Application application)
             }
 
             return ExecuteCore(
+                projectDirectory: application.ProjectRoot.Value() ?? Environment.CurrentDirectory,
```

Comment:
Do this in validate arguments instead. You can manually set the arguments' value using `.Values.Add(...)`